lib/design_system/components/account_summary_card/card_decorations.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/widgets.dart';
```

Comment:
Yes, the intention is that AccountSummaryCard should make 100% use of the design system. This means that in addition to using the design system typography, this component should also use the gradients and icon from the design system rather than using local versions. 

However, when I added the AccountSummaryCard to the Widgetbook, I noticed that it had a few issues that needed to be resolved (I'll show you that tomorrow), so I decided to leave the AccountSummaryCard scope out of this PR and leave the fixing to a separate branch. I did that because I judged that the fixes could easily end up taking a couple of days; Some fiddling to get the UI just right + some stuff that needed to be clarified with Guillermo.